packages/plugin-data/src/dataCommand.ts

```diff
@@ -96,10 +94,17 @@ export abstract class DataCommand extends SfdxCommand {
     return final;
   }
 
+  public ensureOrg(): Org {
+    if (!this.org) {
+      throw new Error('An org is required for a connection');
```

Comment:
@mshanemc I would prefer the error be the same that is emitted when an org cannot be found.

Reply:
done